<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>900</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="worker_portal_tab">
     <attribute name="title">
      <string>Worker Portal</string>
     </attribute>
     <widget class="QPushButton" name="portal3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>380</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="portal4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>380</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="portal2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>130</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="portal1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>70</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Worker:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Worker:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Worker:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>320</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Worker:</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>340</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgb(255, 255, 255);
border-style : solid;
border-color: rgb(170, 255, 255);
border-width : 2px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Current_worker4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgb(255, 255, 255);
border-style : solid;
border-color: rgb(170, 255, 255);
border-width : 2px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Current_worker1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>90</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgb(255, 255, 255);
border-style : solid;
border-color: rgb(170, 255, 255);
border-width : 2px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Current_worker2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgb(255, 255, 255);
border-style : solid;
border-color: rgb(170, 255, 255);
border-width : 2px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Current_worker3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="resistance_tab">
     <attribute name="title">
      <string>Straw Prep</string>
     </attribute>
     <widget class="QGroupBox" name="box_pos_1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>200</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Position 1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLineEdit" name="input_1_ii">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Scan worker ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_1_ii">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>15</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>ii:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="led_1_ii">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_1_ii_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>15</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>ii:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_1_ii_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Scan worker ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_1_ii_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>15</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>ii:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_1_ii_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Scan worker ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_1_ii_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>15</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>ii:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="input_1_ii_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Scan worker ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_1_ii_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>201</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>st0001</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="workerID">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Worker ID</string>
      </property>
      <widget class="QLineEdit" name="workerID_text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Scan worker ID</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="palletID">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Pallet ID</string>
      </property>
      <widget class="QLineEdit" name="palletID_text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Scan pallet ID</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="straw1ID">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Position 1 Straw ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QLineEdit" name="straw1ID_text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Scan 1st straw ID</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="collect_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Collect Data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_button">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>hello_world()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
